spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -143,4 +143,36 @@
       end
     end
   end
+
+  context 'when using sync_multiple_projects_based_on_custom_id mode' do
+    let(:params) do
+      params = {
+        GDC_GD_CLIENT: client,
+        input_source: 'foo',
+        domain: 'bar',
+        filters_config: { labels: [] },
+        multiple_projects_column: 'id_column',
+        sync_mode: 'sync_multiple_projects_based_on_custom_id',
+        gdc_logger: logger
+      }
+      GoodData::LCM2.convert_to_smart_hash(params)
+    end
+    let(:CSV) { double('CSV') }
+
+    before do
```

Comment:
Identical code found in synchronize_user_filters_spec.rb:93, synchronize_user_filters_spec.rb:162 (mass = 152)